### PR TITLE
REL-2988 Release charts for SonarQube 9.9.4

### PR DESCRIPTION
**Pull request by aurelien-poscia-sonarsource**
Please ensure your pull request adheres to the following guidelines:
- [ ] explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Document your Changes in the `CHANGELOG.md` file of the respected chart as well as the `Chart.yaml`

### CODE REVIEW

File: charts/sonarqube-dce/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [8.0.4]
+* Update SonarQube to 
+* 9.9.4
+
 ## [7.0.3]
-* Update SonarQube to 9.9.3
```
Comment (carminevassallo) [inline charts/sonarqube-dce/CHANGELOG.md:5]: This changelog entry should not be modified.

File: charts/sonarqube-dce/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [8.0.4]
+* Update SonarQube to
```
Comment (carminevassallo) [inline charts/sonarqube-dce/CHANGELOG.md:5]: Small typo here, everything should be on same line and we are releasing `7.0.4` instead of `8.0.4`

**Review by carminevassallo — CHANGES_REQUESTED** [review]
Just a small typo to fix in the changelog

File: charts/sonarqube-dce/CHANGELOG.md (hunk 2 above)
Comment (aurelien-poscia-sonarsource) [reply charts/sonarqube-dce/CHANGELOG.md]: outch good catch! and I don't know what happened with formatting, but it is now fixed 👍

File: charts/sonarqube-dce/CHANGELOG.md (hunk 1 above)
Comment (aurelien-poscia-sonarsource) [reply charts/sonarqube-dce/CHANGELOG.md:5]: fixed ✅

**Review by carminevassallo — APPROVED** [review]

**Pull request merged**